examples/getting-started/README.md

```diff
@@ -40,60 +40,36 @@ The behaviour of the order processing back-end, in a nutshell, is shown below.
  
 1. [Download and install the EI tool](https://docs.wso2.com/display/EI6xx/Installing+Enterprise+Integrator+Tooling) that is compliant with the distribution.
  
 2. Create an ESB solution project as explained [here](https://docs.wso2.com/display/EI6xx/Working+with+EI+Tooling#WorkingwithEITooling-CreatinganESBSolutionProjectCreatinganESBSolutionProject) and name it as 'compositeServiceProject'. Deselect the creation of a 'Registry Project' and a 'Connector Exporter Project' since we will not be needing them for this simple project. 
  
 3. Open the project and [create a REST API](https://docs.wso2.com/display/EI6xx/Working+with+EI+Tooling#WorkingwithEITooling-CreatingartifactsforanESBSolutionProject). You can either create an API from scratch and configure it as shown in the following diagram  or directly import the config from [here](composite-service/composite-service/src/main/synapse-config/api/forwardOrderApi.xml).
- 
  ![diagram](images/api-config.png) 
  
-4. Save the project and export the composite application to a desired location. (Alternatively you can build the project 'composite-service' using maven and the C-App will be located in `composite-service/composite-service-capp/target`. The rest of the document will assume that the C-App resides in `composite-service/composite-service-capp/target`
- 
-5. Use docker cli to create the docker image with the both back-end and composite service C-Apps we created.
+4. Save the project and export the composite application to a desired location. (You can refere [How to working with EI Tooling](https://docs.wso2.com/display/EI6xx/Working+with+EI+Tooling#WorkingwithEITooling-PackagingESBartifacts) document for get to know more information about this)
 
-    **note:-** If you need to skip the deployment in docker, you can directly deploy the services locally as explained in the section, [Local Deployment.](#local-deployment) 
-   ```
+**Note -** Alternatively you can build the project 'composite-service' using maven and 'composite-serviceCompositeApplication_1.0.0.car' C-App will be located in `<mi-work-directory>/examples/getting-started/composite-service/composite-service-capp/target` when you use the following command.
+```
    $ cd <mi-work-directory>/examples/getting-started/composite-service
-   $ docker build -t wso2/mi-getting-started .
-   ```
-6. Run the docker image created in the previous step.
-   ```
-   docker run -d -p 8290:8290 wso2/mi-getting-started
+   $ mvn clean install
    ```
 ## Deployment
 
-Through out this section, it will be assumed that you have a C-app representing the services to be deployed.
+Through out this section, it will be assumed that you have created two C-apps for 'composite-service' and 'order-process-backend' which are reside in ```<mi-work-directory>/examples/getting-started/composite-service/composite-service-capp/target``` and ```<mi-work-directory>/examples/getting-started/order-process-be/order-processCompositeApplication/target``` folders.
```

Comment:
```suggestion
Throughout this section, it will be assumed that you have created two C-apps for 'composite-service' and 'order-process-backend' which are reside in ```<mi-work-directory>/examples/getting-started/composite-service/composite-service-capp/target``` and ```<mi-work-directory>/examples/getting-started/order-process-be/order-processCompositeApplication/target``` folders.
```

Reply:
Done